src/main/default/classes/BaseTriggerTest.cls

```diff
@@ -0,0 +1,17 @@
+@isTest
+public class BaseTriggerTest {
+  @isTest
+  private static void testInMemoryBaseClass() {
```

Comment:
test ergonomics look great

Reply:
I guess these in memory tests would pretty much only be for `before update`?

In theory you could test other events, but you'd have to generate valid `Id`s (which probably isn't hard) and you would 't be able to test recursion...